repo_resource/test_check.py

```diff
@@ -306,14 +315,14 @@ def test_ssh_private_key_without_manifest_access(self):
         with self.assertRaises(SystemExit):
             versions = check.check(instream)
 
-        self.assertEquals(len(versions), 0)
```

Comment:
These seem irrelvant to the commit and are solved in:

https://github.com/makohoek/repo-resource/pull/29

I will merge the above PR.